src/index.html

```diff
@@ -65,7 +65,17 @@ <h1 class="header">Project 1</h1>
             <img src="https://via.placeholder.com/200x200" alt="project image">
             <img src="https://via.placeholder.com/200x200" alt="project image">
           </div>
-        </1div>
+          <hr size="4" align="center" color="#666262">
+          <h1 class="header">Project 2</h1>
```

Comment:
Each project should be contained inside a `div class="project"></div>` and the separator should be between each `.project` div.